src/scripts/towns/TownList.ts

```diff
@@ -2343,7 +2343,7 @@ const FortreeRanger = new NPC('Pokémon Ranger Catherine', [
     'Have you seen this? It\'s horrific. Absolutely despicable.',
     'Disposable Dowsing Machines! Who even came up with this?! People leave these everywhere. It\'s absolutely terrible for the environment. Some poor innocent Pokémon could choke on it!',
     'Please recycle your used Dowsing Machines.',
-], {image: 'assets/images/npcs/Pokemon Ranger (female).png'});
+], {image: 'assets/images/npcs/Pokémon Ranger (female).png'});
```

Comment:
These aren't typos, but deliberate spellings. We have two female Ranger sprites, differentiated by that `é`.